ui/components/app/app-header/app-header.component.js

```diff
@@ -9,7 +9,7 @@ import NetworkDisplay from '../network-display';
 
 ///: BEGIN:ONLY_INCLUDE_IN(beta)
 import BetaHeader from '../beta-header';
-///: END:ONLY_INCLUDE_IN(beta)
+///: END:ONLY_INCLUDE_IN
```

Comment:
why were these changes made?

Reply:
I was attempting to remove the custom code fencing transform alltogether, but as legobeat pointed out its probable that my changes were not fully effective.